src/services/consumer.js

```diff
@@ -57,7 +57,7 @@ class Consumer {
                                 continue;
                             }
                             const gym = Gym.fromFort(fort.cell, fort.data);
-                            await gym.triggerWebhook();
+                            gym.triggerWebhook();
```

Comment:
```suggestion
                            await gym.triggerWebhook();
```
We need await for promises